packages/phoenix-client/src/phoenix/client/types/spans.py

```diff
@@ -1,6 +1,40 @@
 from dataclasses import dataclass
 from dataclasses import field as dataclass_field
-from typing import Any, Optional
+from datetime import datetime, timezone
+from typing import Any, Optional, Union
+
+from typing_extensions import NotRequired, TypedDict
+
+
+class SpanStatus(TypedDict):
+    code: NotRequired[str]  # "UNSET", "OK", "ERROR"
+    message: NotRequired[str]
+
+
+class SpanEvent(TypedDict):
+    name: NotRequired[str]
+    timestamp: NotRequired[datetime]
+    attributes: NotRequired[dict[str, Any]]
+
+
+class Span(TypedDict):
```

Comment:
I think it makes sense to push this into the openapi schema since the type safety will be nice. We can always do v2 spans down the line. Can we mirror it against this format? https://opentelemetry.io/docs/concepts/signals/traces/